engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ExecutionStoreBenchmark.scala

```diff
@@ -0,0 +1,52 @@
+package cromwell.engine.workflow.lifecycle.execution
```

Comment:
Yeah as awesome as it'd be to have this sort of thing in our CI I think just having "here's how one could run this and here is what it is measuring" in the comments would suffice for now :)